website/more_documentation/button_documentation.py

```diff
@@ -1,4 +1,9 @@
+from asyncio import sleep
```

Comment:
These imports should move into `disable_context_manager` so that they are shown as part of the example.

Reply:
Done